mindsdb_sql_parser/ast/table.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional, List, Any
+
+from mindsdb_sql_parser.ast.base import ASTNode
+
+
+class Table(ASTNode):
```

Comment:
`table a` is the same as `select * from a`, why to not parse it as CreateTable?

Reply:
I guess you mean `Select`.
Indeed, `select a` and `select * from a` are equivalent. I prefer to have separate `Table` class because `Select` and `Table` are separate statements. However, it would be much easier to render this statement to different dialects if it will be `Select`, so i changed it.